config/default.js

```diff
@@ -335,5 +335,6 @@ module.exports = {
   // experiments.
   experiments: {
     home_hero: true,
+    install_button_color: true,
```

Comment:
Is there a reason why these aren't camel case?

Reply:
The only reason I can provide is that this ID is used in the cookie name, which is prettier all in snake case.